README.md

```diff
@@ -226,8 +226,8 @@ ad_defaults:
 # additional name to category ID mappings, see default list at
 # https://github.com/Second-Hand-Friends/kleinanzeigen-bot/blob/main/kleinanzeigen_bot/resources/categories.yaml
 categories:
- #Notebooks: 161/27
- #PCs: 161/228
+ #Elektronik > Notebooks: 161/278
+ #Auto, Rad & Boot > Autoteile & Reifen > Weitere Autoteile: 210/223/sonstige_autoteile
```

Comment:
categories is actually a key-value hashmap in the format `key: value`
If your goal was to use special characters as part of the key they would better be quoted. E.g.
```yaml
categories:
   "Auto, Rad & Boot > Autoteile & Reifen > Weitere Autoteile": 210/223/sonstige_autoteile
```

By registering a category like this you will then however have reference this long value in every add. 

Maybe adding  comments like this would be better:
```yaml
categories:
   # autoteile: 210/223/sonstige_autoteile # Auto, Rad & Boot > Autoteile & Reifen > Weitere Autoteile
```
Now you only have to remove the leading hash symbol to have a valid entry and still have the full path at the end of the line as a comment.

Reply:
My goal was to demonstrate the usage of the 3rd level, but I had only documentation in mind. A comment after the `key: value` pair is the best way IMHO. Fixed in [fee23f9](https://github.com/Second-Hand-Friends/kleinanzeigen-bot/pull/23/commits/fee23f9f05958d3dc87a09bd73d5b2838bdb96e9).